packages/react/src/components/Table/TableBody/TableBodyRow/TableBodyRow.jsx

```diff
@@ -381,6 +381,7 @@ const StyledExpansionTableRow = styled(({ hasRowSelection, ...props }) => <Table
 const StyledNestedSpan = styled.span`
   position: relative;
   left: ${(props) => props.nestingOffset}px;
+  max-width: calc(100% - ${(props) => props.nestingOffset}px);
```

Comment:
We should not be doing this with styled components. This prop should be added as a css custom property.